### DISCUSSION TITLE
Rotate the profile photo in mobile version

### DISCUSSION THREAD

**Discussion by bertux77** [livewire/livewire#3489] [1. Help]
I am using laravel jetstream with livewire and enabled Features::profilePhotos().

When I upload a profile picture from the computer, it displays and saves it correctly but if I upload it from the mobile it rotates the picture.

@if ($photo)
	Photo Preview:
       	`<img` src="{{ $photo->temporaryUrl() `}}">
@endif
How can this be solved?

**Comment by joshhanley** [comment]
@bertux77 maybe the orientation of the photo on your phone is rotated and your phone is correcting it in your photos app?

Can you test with some other photos and see if it still happens?

Also what version of Livewire and Laravel are you using?
